packages/components/pie-chip/src/index.ts

```diff
@@ -92,10 +96,15 @@ export class PieChip extends LitElement implements ChipProps {
 
         return html`
             <div
+                aria-atomic="true"
+                aria-busy="${isLoading ? 'true' : 'false'}"
```

Comment:
Would this work? The lack of leading `?` might mean that it ends up being a string and not a boolean
```suggestion
                aria-busy="${isLoading}"
```

Reply:
the aria-busy value is a true or false string not a boolean 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy

Reply:
Yeah, exactly! For a boolean we would do `?aria-busy=${isLoading}` so I'm wondering if we can just do `aria-busy=${isLoading}` to get the string value.